### ISSUE TITLE
Are Pattern specific functions supported?

### ISSUE THREAD

**Issue by AhmedSalahKhodair** [christianrondeau/SikuliSharp#22]
Functions such as `Pattern.exact()`, `Pattern.anySize()` or `Pattern.anyColor()`.

**Comment by christianrondeau** [comment]
Nope, but it should be pretty simple to add support for this! Let me know if you'd like information on this (make sure you can compile and run from a cloned version first)
